ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-hardware.j2

```diff
@@ -0,0 +1,26 @@
+{% if ip_hardware is arista.avd.defined %}
+
+## IP Hardware
+{%   if ip_hardware.fib is arista.avd.defined %}
+
+### FIB
+{%     if ip_hardware.fib.optimize is arista.avd.defined %}
+
+#### Optimize
+{%       if ip_hardware.fib.optimize.prefix_profile is arista.avd.defined %}
+
+##### Prefix Profile
+
+| Profile |
+| ------- |
+| {{ ip_hardware.fib.optimize.prefix_profile }} |
+
+### IP Hardware Device Configuration
```

Comment:
line 18-22 are independent of the prefix_profile (Currently this hardware device config will be printed out only if prefix_profile is defined). Let's say in future someone adds a new knob under FIB. So please keep this section under ip_hardware if condition.

Reply:
Similarly for the other template.